frontend/src/scenes/userLogic.tsx

```diff
@@ -40,6 +41,12 @@ export const userLogic = kea<userLogicType<UserType, UpdateUserPayload>>({
                 setUserLoading: (_, { loading }) => loading,
             },
         ],
+        productionEnvironment: [
```

Comment:
I think it's better to keep this logic in `navigationLogic`, as it's related to those components and not user-scoped. I would also not treat this as a boolean state but set the two options explicitly (for code clarity purposes, it also avoids confusion on later implementations). Something like: `type GlobalEnvironment = 'live' | 'test'`